File: src/main/java/ru/arc/hooks/PlayerWarpsHook.java
package ru.arc.hooks;

import com.olziedev.playerwarps.api.PlayerWarpsAPI;
import com.olziedev.playerwarps.api.warp.Warp;
import org.bukkit.entity.Player;

import static ru.arc.util.Logging.info;

public class PlayerWarpsHook {

    public boolean warpExists(String name, Player player){
        Warp warp = PlayerWarpsAPI.getInstance().getPlayerWarp(name, player);
        info("Warp name {} {}", name, warp);
        return warp != null;
    }

}

File: src/main/kotlin/ru/arc/hooks/PlayerWarpsHook.kt
package ru.arc.hooks

import com.olziedev.playerwarps.api.PlayerWarpsAPI
import org.bukkit.entity.Player
import ru.arc.util.Logging.info

class PlayerWarpsHook {
    fun warpExists(name: String, player: Player): Boolean {
        val warp = PlayerWarpsAPI.getInstance().getPlayerWarp(name, player)
        info("Warp name {} {}", name, warp)
        return warp != null
    }
}
